src/FormMigration/Steps/ConstantContact.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Give\FormMigration\Steps;
+
+use Give\FormMigration\Contracts\FormMigrationStep;
+use Give\Framework\Blocks\BlockModel;
+
+/**
+ * @unreleased
+ */
+class ConstantContact extends FormMigrationStep
+{
+    /**
+     * @unreleased
+     */
+    public function canHandle(): bool
```

Comment:
Let's create unit tests for these

Reply:
Hey Jon, I've added the tests for this. I've also moved the global options into the default prop for the getMeta method. Resolved: 16ab4da1e2f98a94a99464019f9847ba6baac6fc & df77f5c8d89b352ae99644a890efa07f3e6b6c1f

Additionally while rearranging the CC methods to all be together some of the tags got mixed between them & Mailchimp. Ive fixed them in this commit : 607a2bac0520fcd2c82442b14ff6b88131f4c41f